src/electron/platform/android/scan-controller.ts

```diff
@@ -73,8 +73,8 @@ export class ScanController {
         const payload = this.unifiedResultsBuilder(data);
 
         this.unifiedScanResultAction.scanCompleted.invoke(payload);
-        this.scanActions.scanCompleted.invoke(null);
-        this.deviceConnectionActions.statusConnected.invoke(null);
+        this.scanActions.scanCompleted.invoke();
```

Comment:
passing `null` vs `void` does not alter the behavior of the invocation so we should call the function without params as intended